idd/Energy+.idd.in

```diff
@@ -67346,7 +67346,7 @@ ThermalStorage:ChilledWater:Stratified,
        \type node
   N13, \field Source Side Heat Transfer Effectiveness
        \type real
-       \minimum 0.0
+       \minimum> 0.0
```

Comment:
OK.  That's kind of surprising, I would have expected both sides to use effectiveness in similar types of calculations.